src/Modal.js

```diff
@@ -55,7 +55,7 @@ const Modal = React.createClass({
         className={classNames(this.props.className, classes)}
         onClick={this.props.backdrop === true ? this.handleBackdropClick : null}
         ref="modal">
-        <div className={classNames(dialogClasses)}>
+        <div className={classNames(this.props.dialogClassName, dialogClasses)}>
```

Comment:
I think adding tests around a React.PropType validator is overkill. Our linter should have picked up that a new prop type was getting used without being declare though... @AlexKVal Have you seen this issue with the react eslint plugin not picking that up?